_data/committee/members.yml

```
@@ -400,9 +401,22 @@
   email: nooriyah.lohani@nesi.org.nz
   affiliation: New Zealand eScience Infrastructure
   location: Auckland, NZ
+  avatar: /assets/images/Nooriyah-Lohani.jpg
   twitter: MysteryGenes
   github: nooriyahp
   orcid: 0000-0002-2051-4821
   teams:
     - aunzRSE
     - orga
+- name: Simon Hettrick
+  email: s.hettrick@software.ac.uk
+  website: https://www.linkedin.com/in/simon-hettrick-2668a9a/
+  affiliation: Software Sustainability Institute
+  location: Southampton, UK
+  avatar: /assets/images/SimonHettrick.jpg
+  twitter: sjh5000
+  github: SimonHettrick
+  orcid: 0000-0002-6809-5195
+  teams:
+    - orga
+    - ukRSE
```

```suggestion
    - ukRSE
    - orga
```
for the others we always have the order

1. national chapter
2. committee
3. programme teams